scss/pivotgrid/_theme.scss

```diff
@@ -0,0 +1,66 @@
+@include exports('pivotgrid/theme') {
+    $pivotgrid-alt-bg: $header-bg !default;
+    $pivotgrid-chrome-border: $border-color !default;
+
+    $pivotgrid-container-bg: $bg-color !default;
+    $pivotgrid-row-headers-bg: darken($bg-color, 7%) !default;
+
+    .k-alt,
+    .k-pivot-toolbar,
+    .k-pivot-layout > tbody > tr:first-child > td:first-child {
+        background-color: $pivotgrid-alt-bg;
+    }
+
+    .k-fieldselector .k-list-container {
+        background-color: $pivotgrid-container-bg;
+    }
+
+    .k-pivot-toolbar,
+    .k-pivot-table,
+    .k-fieldselector .k-list-container,
+    .k-fieldselector .k-columns > div,
+    .k-pivot-rowheaders > .k-grid td:first-child,
+    .k-pivot-table .k-grid-header .k-header.k-first {
+        border-color: $pivotgrid-chrome-border;
+    }
+
+    .k-pivot-rowheaders .k-alt .k-alt,
+    .k-header.k-alt {
+        background-color: $pivotgrid-row-headers-bg;
+    }
+
+    .k-pivot-toolbar .k-button,
+    .k-fieldselector .k-list li.k-item {
+        @include appearance( button );
+
+        &:hover,
+        &.k-state-hover {
+            @include appearance( hovered-button );
+        }
+
+        // Pressed state
+        &:active,
+        &.k-state-active {
+            @include fill( $button-text, $button-bg, $button-border, #{to top, $button-gradient} );
```

Comment:
Does this match the pressed button state? If so, why no use `@include appearance( pressed-button );`?